.github/workflows/code-coverage.yml

```
@@ -5,37 +5,42 @@ on:
-        uses: codecov/codecov-action@v5.5.2
-        with:
-          token: ${{secrets.CODECOV_TOKEN}}
+    runs-on: ubuntu-latest
 
-      - name: Archive code coverage results
-        uses: actions/upload-artifact@v6
-        with:
-          name: code-coverage-report
-          path: cobertura.xml
+    steps:
+    - uses: actions/checkout@v3
```

actionlint (1.7.10)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)